python-backend/app/api/permits/permit/resources/permit.py

```diff
@@ -86,6 +100,17 @@ def post(self, permit_guid=None):
                 description='Initial permit issued.')
             amendment.save()
             permit.save()
+
+            for newFile in uploadedFiles:
+                new_pa_doc = PermitAmendmentDocument(
+                    document_name=newFile['fileName'],
+                    document_manager_guid=newFile['document_manager_guid'],
+                    mine_guid=permit.mine_guid,
+                    **self.get_create_update_dict(),
+                )
+                amendment.documents.append(new_pa_doc)
+            amendment.save()
+            permit.save()
```

Comment:
We're saving these on lines 101-102 as well. Is that intentional?

Reply:
Changed, should make more sense now.